week2/homework/src/reset.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+/** ---------------------------------------
+     Base
+ ---------------------------------------- */
+const fs = require('fs');
+
+/** ---------------------------------------
+     Function for Command
+ ---------------------------------------- */
+const reset = {
+  execute: () => {
+    const content = fs.readFileSync('To-Do Database.json', 'utf8');
+    switch (content) {
+      case '':
+        console.log('There is nothing to remove !');
+        break;
+      default:
+        fs.writeFile('To-Do Database.json', '', err => {
```

Comment:
In alternative of `''`, you could write an empty array: `'[]'`.
Doing that, you don't need to use fillEmptyDatabase in the add functionality

Reply:
But the client doesn't call 'reset' before each command. Also, the client always start with an empty database. Therefore, 'add' command always check for if it is an empty database.